v3/src/components/slider/editable-slider-value.tsx

```diff
@@ -38,8 +39,8 @@ export const EditableSliderValue = observer(function EditableSliderValue({ slide
     const inputValue = parseFloat(e.target.value)
     if (isFinite(inputValue)) {
       sliderModel.encompassValue(inputValue)
-      setCandidate(multiScale.formatValueForScale(sliderModel.value))
       sliderModel.setValue(inputValue)
+      setCandidate(multiScale.formatValueForScale(sliderModel.value))
```

Comment:
This might not be necessary with the above change because the autorun will take care of it.